EIPS/eip-6269.md

```diff
@@ -0,0 +1,58 @@
+---
+eip: 6269
+title: Full EVM Equivalence
+description: Canonicalise the definition of Full EVM Equivalence
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/evm-equivalence-and-ethereum-stack-compatibility-definition-future-informational-eip/10044
+status: Draft
+type: Informational
```

Comment:
```suggestion
type: Meta
```
Since this proposes changes that cannot be ignored, I suggest that this be a Meta-EIP.

Reply:
Makes sense